force-app/main/lwcComponentLibrary/lwc/singleLookup/singleLookup.html

```diff
@@ -0,0 +1,113 @@
+<template>
+    <div class="slds-form-element">
+        <label id="lookupLabel" class="slds-form-element__label" if:true={label} for="lookupInput">
+            <abbr class="slds-required" if:true={required}>*</abbr>{label}
+        </label>
+        <div class="slds-form-element__control">
+            <div class={containerClass}>
+                <div class={comboboxClass} aria-expanded={isExpanded} aria-haspopup="listbox" role="combobox">
```

Comment:
I actually like how this html is handling classes. Not consistent with other components thou. We might need to clean up at a certain point of time.

Reply:
This is a slightly different style, but it may be a good pattern to apply elsewhere. Since this was later work and I learned from looking at some SAL code, the way it functions is a bit more clean.